docs/hadoop-catalog.md

```diff
@@ -125,14 +132,15 @@ Refer to [Schema operation](./manage-fileset-metadata-using-gravitino.md#schema-
+|---------------------------------------|----------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------------|-----------|------------------|
+| `authentication.impersonation-enable` | Whether to enable impersonation for the Hadoop catalog fileset.                                                      | The parent(schema) value                                                                                       | No                                         | Yes       | 0.6.0-incubating |
+| `authentication.type`                 | The type of authentication for Hadoop catalog fileset, currently we only support `kerberos`, `simple`.               | The parent(schema) value                                                                                       | No                                         | No        | 0.6.0-incubating |
+| `authentication.kerberos.principal`   | The principal of the Kerberos authentication for the fileset.                                                        | The parent(schema) value                                                                                       | No                                         | No        | 0.6.0-incubating |
+| `authentication.kerberos.keytab-uri`  | The URI of The keytab for the Kerberos authentication for the fileset.                                               | The parent(schema) value                                                                                       | No                                         | No        | 0.6.0-incubating |
+| `credential-providers`                | The credential provider types, separated by comma.                                                                   | (none)                                                                                                         | No                                         | No        | 0.8.0-incubating |
+| `placeholder-`                        | Properties that start with `placeholder-` are used to replace placeholders in the location.                          | (none)                                                                                                         | No                                         | Yes       | 0.9.0-incubating |
+| `default-location-name`               | The name of the default location of the fileset, mainly used for GVFS operations without specifying a location name. | When the fileset has only one location, its location name will be automatically selected as the default value. | Yes, if the fileset has multiple locations | Yes       | 0.9.0-incubating |
```

Comment:
This is an example why Markdown table is not suitable for wide tables.
Every time we add a new row to the table, the whole table is changed.
It is hard to tell what the real changes are.

Reply:
Any suggestion to improve this?

Reply:
I'd suggest we use HTML tables directly for "wide" tables.

Reply:
Sounds good, we can track this by another issue and PR since this is a major change for doc.

Reply:
Right. Please DON'T do it in this PR. I'll take care of all the conversions in another PR (#6849).